docs/api/console.md

```diff
@@ -0,0 +1,159 @@
+# 🎮 Console
+
+An implementation of the standard `console` API.
+
+## Quick start
+
+### Log messages
+
+`Console` provides the following general purpose logging functions.
+* `console.log` for general messages.
+* `console.info` for information messages.
+* `console.warn` for warnings.
+* `console.error` for errors.
+* `console.debug` for debug level messages.
```

Comment:
nit: not too sure we need the description of the functions, the names are self explanatory and the method descriptions below elaborate more